lib/token-assert.js

```diff
@@ -143,7 +143,7 @@ TokenAssert.prototype.differentLine = function(options) {
 TokenAssert.prototype.linesBetween = function(options) {
     var token = options.token;
     var nextToken = options.nextToken;
-    var whitespaceBefore = ' ';
+    var whitespaceBefore = '';
```

Comment:
That is definitely a valid case. It makes me sad though that we are adding a space on rules that only care about lines. :-/

I'll change this back.